src/demo/components/Graphs.tsx

```diff
@@ -151,6 +153,14 @@ const Graphs: any = (props: {
     )
   }
 
+  const sevenDayAveData = movingAveDataSort(Object.values(data))
+
+  const getSevenDayAveData = (data: Data): number => {
+    const index = sevenDayAveData.findIndex(obj => obj.date === data.date)
+    if (typeof index !== 'number' || isNaN(index)) return 0
+    return sevenDayAveData[index].allTxs
+  }
```

Comment:
This looks like it could be unit tested.

Reply:
This function currently relies on two sources of data, and I am not sure it is possible to pass both as function params given how recharts is currently set up.  That is why it is not currently tested.

Reply:
Data flow was adjusted so this function was changed and is now being tested.